### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by mplachter**
# Changes for Branch
---------------------
* Restructured project to make idempotence
  * Broke up condition tasks to handlers
  * Made Configure Ticks File
* Fixed tests
  * Added Travis File
     * Removed gitlab ci
  * Added More Tests
* Passing Travis Build
  * https://github.com/mplachter/ansible-role-kapacitor
    * Created new branch so travis badge would be from current repo CI

No installation/configuration changes were made. 
This was just to get the CI working.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,6 +0,0 @@
----
```
Comment (popstas) [inline .gitlab-ci.yml]: I need gitlab-ci for vagrant full test

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+sudo: required
+language: python
+services:
+  - docker
+before_install:
+  - sudo apt-get -qq update
+install:
+  - pip install ansible
```
Comment (popstas) [inline .travis.yml]: Molecule installs ansible as dependency, not necessary to install it explicitly

File: tasks/configure-thresholds.yml [hunk 3]
```diff
@@ -6,19 +6,9 @@
 
 - name: Build stream_threshold TICK
   template:
+    force: yes
```
Comment (popstas) [inline tasks/configure-thresholds.yml]: Why force?

**Review by popstas — CHANGES_REQUESTED** [review]
Thank you, found some new ansible features and use cases of molecule from your PR ) Leaved some comments at code, please check. 

Only one request: please don't delete `.gitlab-ci.yml`.

I hope that I will test it tomorrow on real machine, then PR will be merged.

File: .travis.yml (hunk 2 above)
Comment (mplachter) [reply .travis.yml]: I agree, this is redundant.

File: tasks/configure-thresholds.yml (hunk 3 above)
Comment (mplachter) [reply tasks/configure-thresholds.yml]: Force is default yes this is redundant.

File: .gitlab-ci.yml (hunk 1 above)
Comment (mplachter) [reply .gitlab-ci.yml]: Sorry about that, i will add back the gitlab-ci.

**Review by popstas — APPROVED** [review]

**Comment by popstas** [comment]
Good! I tested role on real machine with Ubuntu 16.04, all works fine! What about squash commits from 20+ to 1-5? I ready to merge as is, but I think, that your commits kind of... verbose. What you think about it?

**Comment by mplachter** [comment]
Yeah i can squash them 👍

**Comment by popstas** [comment]
Good job, thank you!

**Pull request merged**